frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/strings/storage.ts

```diff
@@ -1,3 +1,5 @@
 export const ATTACH_CD = 'Attach CD-ROM';
-export const NO_BOOTABLE_ATTACHED_DISK_ERROR = 'A bootable attached disk could not be found';
+export const BOOTABLE_ATTACHED_DISK_MESSAGE = 'Boot disks can not have a blank source';
```

Comment:
blank source has a special meaning. Shouldn't we say something like that `selecting a boot source is required`?

Reply:
@matthewcarleton can you jump in?

Reply:
What we want to communicate is that the disk they are selecting must have a source defined and can not be blank. 
Could we say:
"A boot source disk must have a source and can not be blank"

Reply:
not sure if we want to mention other invalid sources (which is not just blank type), but ok let's put this message